Detector/DetSensitive/src/SimpleTrackerSD.cpp

```diff
@@ -53,9 +51,7 @@ bool SimpleTrackerSD::ProcessHits(G4Step* aStep, G4TouchableHistory*)
     track->GetTrackID(), track->GetDefinition()->GetPDGEncoding(),edep, track->GetGlobalTime());
   // hit is expected to be created, otherwise abort job
   assert(hit != nullptr);
-  // take mid position between prestep and poststep to calculate the position in segmentation,
-  // to compare output hits with DD4hep Geant4Tracker
-  hit->cellID  = segmentation::cellID(m_seg, *aStep, false);
+  hit->cellID  = segmentation::cellID(m_seg, *aStep);
```

Comment:
as explained [here](https://github.com/zaborowska/FCCSW/blob/Dd4hepCells/Detector/DetSensitive/DetSensitive/SegmentationHelper.h#L15) it's a flag introduced so that we may choose not to use the default position at G4Step::GetPreStepPoint and instead use middle between G4Step::GetPreStepPoint and G4Step::GetPostStepPoint, as it is in DD4hep. I left that possibility for a) test purposes b) maybe sb will want to do that? It's used in the other SD, `MiddleStepTrackerSD`